ibis/backends/dask/helpers.py

```diff
@@ -0,0 +1,186 @@
+from __future__ import annotations
+
+from typing import Callable
+
+import dask.array as da
+import dask.dataframe as dd
+import numpy as np
+import pandas as pd
+
+from ibis.backends.pandas.helpers import PandasUtils
+
+
+class DaskUtils(PandasUtils):
+    @classmethod
+    def merge(cls, *args, **kwargs):
+        return dd.merge(*args, **kwargs)
+
+    @classmethod
+    def merge_asof(cls, *args, **kwargs):
+        return dd.merge_asof(*args, **kwargs)
+
+    @classmethod
+    def concat(cls, dfs, **kwargs):
+        if isinstance(dfs, dict):
+            dfs = [v.rename(k) for k, v in dfs.items()]
+        return dd.concat(dfs, **kwargs)
+
+    @classmethod
+    def asseries(cls, value, like=None):
+        """Ensure that value is a pandas Series object, broadcast if necessary."""
+
+        if isinstance(value, dd.Series):
+            return value
+        elif isinstance(value, dd.core.Scalar):
+            # Create a Dask array from the Dask scalar
+            try:
+                dtype = value.dtype
+            except AttributeError:
+                #     @property
+                #     def dtype(self):
+                # >       return self._meta.dtype
+                # E       AttributeError: 'Timestamp' object has no attribute 'dtype'
```

Comment:
Is this left over debugging code, or is there on purpose to show what triggers the `AttributeError`?

Reply:
The latter, going to leave it here for now so someone can fix it.